plugins/grpc-ecosystem/gateway/v2.15.2/separate_pkg_additional_imports.patch

```diff
@@ -342,26 +341,19 @@ index 023a18e3..732132f2 100644
  	generateUnboundMethods     = flag.Bool("generate_unbound_methods", false, "generate proxy methods even for RPC methods that have no HttpRule annotation")
 +	separatePackage            = flag.Bool("separate_package", false, "generate gateway code to v1gateway package (requires standalone=true).")
  )
-
+ 
  // Variables set by goreleaser at build time
-@@ -62,9 +64,11 @@ func main() {
+@@ -62,9 +64,21 @@ func main() {
  			return err
  		}
-
--		codegenerator.SetSupportedFeaturesOnPluginGen(gen)
+ 
 +		if *separatePackage && !*standalone {
 +			return errors.New("option separate_package=true must be specified with standalone=true")
 +		}
-
++
+ 		codegenerator.SetSupportedFeaturesOnPluginGen(gen)
+ 
 -		generator := gengateway.New(reg, *useRequestContext, *registerFuncSuffix, *allowPatchFeature, *standalone)
-+		codegenerator.SetSupportedFeaturesOnPluginGen(gen)
-
- 		glog.V(1).Infof("Parsing code generator request")
-
-@@ -86,6 +90,15 @@ func main() {
- 			targets = append(targets, f)
- 		}
-
 +		generator := gengateway.New(
```

Comment:
This is the fix for the import alias issue - moved the declaration of the generator higher up so this code can run earlier: https://github.com/grpc-ecosystem/grpc-gateway/blob/7da715e510ff64375596028d4e5108d6255ceb01/protoc-gen-grpc-gateway/internal/gengateway/generator.go#L31-L59